src/pages/components/tooltip/code.mdx

```diff
@@ -1,83 +1,96 @@
 ---
 title: Tooltip
 description: Tooltips provide additional information upon hover or focus. The information should be contextual, useful, and nonessential information.
-tabs: ['Code', 'Usage', 'Style', 'Accessibility']
+tabs: ["Code", "Usage", "Style", "Accessibility"]
 ---
 
-import { Filter16 } from '@carbon/icons-react';
-
-## Tooltip
-
-<ComponentDemo
-  links={{
-    React:
-      'https://react.carbondesignsystem.com/?path=/story/tooltip--default-bottom',
-    Angular:
-      'https://angular.carbondesignsystem.com/?path=/story/components-tooltip--basic',
-    Vue:
-      'http://vue.carbondesignsystem.com/?path=/story/components-cvtooltip--default-interactive-tootlip',
-    Vanilla: 'https://the-carbon-components.netlify.com/?nav=tooltip',
-  }}>
-  {`
-  <Tooltip
-    direction="bottom"
-    tabIndex={0}
-    triggerText="Tooltip label"
-  >
-    <p>
-      This is some tooltip text. This box shows the maximum amount of text that should appear inside. If more room is needed please use a modal instead.
-    </p>
-    <div className="bx--tooltip__footer">
-      <Link href="#">Learn More</Link>
-      <Button size="small">
-        Create
-      </Button>
-    </div>
-  </Tooltip>
-  `}
-</ComponentDemo>
-
-## Icon tooltip
+import { Filter16 } from "@carbon/icons-react";
 
 <ComponentDemo
```

Comment:
Theming bug, default tooltip popup not rendering correctly in Gray 90 and Gray 100 themes. Icon and definition tooltip are correct. 

Is showing:
![image](https://user-images.githubusercontent.com/11670886/78407778-2eab2380-75cb-11ea-810e-82627e20fce0.png)

Should be:
![image](https://user-images.githubusercontent.com/11670886/78407730-1804cc80-75cb-11ea-8964-97ff6ec5bca8.png)